scrapers/MyDirtyHobby.yml

```
@@ -12,6 +12,11 @@ xPathScrapers:
       Date:
         selector: //dd[i[contains(@class,"fa-calendar")]]
         postProcess:
+          - replace:
+            - regex: ^(today).+
+              with: today
+            - regex: ^(yesterday).+
+              with: yesterday
```

Maybe it's just me, but I like to see as few replacements as possible... 😀
```suggestion
            - regex: ^(today|yesterday).+
              with: $1
```

---

No you are absolutely right. I was using the scraper to test the relevant stash PR and forgot to finish it(hence the unused capture groups and separate regex). Thanks for noticing